x-pack/plugin/ent-search/build.gradle

```diff
@@ -19,3 +19,8 @@ dependencies {
 }
 
 addQaCheckDependencies(project)
+
+testClusters.configureEach {
+  setting 'xpack.ent-search.enabled', 'true'
+}
```

Comment:
Why is this needed?

Reply:
I guess we will need the feature toggle enabled to perform the API testing - I included it here for the subsequent testing work, but I can remove it and add it later.

Reply:
The config for the test clusters is [here](https://github.com/elastic/elasticsearch/blob/ent-search-module/x-pack/plugin/ent-search/qa/rest/build.gradle). 
We don't have rest tests outside of `qa`.

Reply:
Got it, removing that!